backend/app/api/daily_journal.py

```
@@ -0,0 +1,275 @@
+        return None
+    if include_revisions:
+        # revisions 관계 로드
+        dj = await daily_journal_store.get_by_id(
+            db, current_user.id, dj.id, include_revisions=True
+        )
+    return _to_response(dj, include_revisions=include_revisions)
+
+
+@router.get("/export-pdf")
+async def export_period_pdf(
+    date_from: date = Query(...),
```

Ruff (0.15.10)

[warning] Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)